FirebaseAuthUI/FUIPrivacyAndTermsOfServiceView.m

```diff
@@ -38,79 +38,53 @@ - (void)useFooterMessage {
   self.textAlignment = NSTextAlignmentRight;
 }
 
-#pragma mark - Private
+#pragma mark - Protected
 
-- (NSAttributedString *)fullPrivacyPolicyAndTOSMessage {
-  if (![FUIAuth defaultAuthUI].TOSURL.absoluteString.length &&
-      ![FUIAuth defaultAuthUI].privacyPolicyURL.absoluteString.length) {
+- (NSAttributedString *)privacyPolicyAndTOSMessageFromFormat:(NSString *)format {
+  FUIAuth *authUI = self.authUI;
```

Comment:
Consider changing this line to

```objc
FUIAuth *authUI = self.authUI ?? [FUIAuth defaultAuthUI];
```

in case the authUI instance isn't set when this method is called.

Reply:
Done.